CHANGES.txt

```diff
@@ -12,6 +12,10 @@ RELEASE  VERSION/DATE TO BE FILLED IN LATER
   From Joseph Brill:
     - Verify that a user specified msvc script (via MSVC_USE_SCRIPT) exists and raise an exception
       when the user specified msvc script does not exist.
+    - Replace non-existent MSVS environment variables VS70COMNTOOLS and VS60COMNTOOLS with 
```

Comment:
This is a very internal developer focused blurb. Can you add more detail on what this would fix from user perspective?  I'm guessing something like "Fixes issue initializing SCons when useing MSVC 6.0 -> 7.0" ? And maybe a little details?  (same for RELEASE.txt)

Reply:
I pushed a new CHANGES.txt and RELEASE.txt. Are the updated blurbs heading in the right direction?